scripts/generate_pkcs7_tests.py

```diff
@@ -37,7 +38,13 @@ class TestData:
     Take in test_suite_pkcs7.data file.
     Allow for new tests to be added.
     """
-    mandatory_dep = "PSA_WANT_ALG_SHA_256"
+
+    #temporary solution to determine correct dependency macrosbetween 3.6 and 4.0
```

Comment:
missing space between `macros` and `between`

Reply:
Otherwise please reference the issue #51 and probably better to rebase your two commits on top of main.